astroquery/cadc/core.py

```diff
@@ -596,6 +617,9 @@ def exec_sync(self, query, maxrec=None, uploads=None, output_file=None):
             Temporary tables to upload and run with the queries
         output_file: str or file handler:
```

Comment:
nitpick: add spaces before `:`, here and elsewhere, to make this rendered properly

Reply:
Done